### DISCUSSION TITLE
issues category

### DISCUSSION THREAD

**Discussion by rootsmusic** [community/community#28096] [Other Feature Feedback, Questions, & Ideas] [answered]
Is the "[issues](https://github.com/orgs/community/discussions/categories/issues)" category for reporting technical issues about Github?  Or is it about [https://github.com/issues](https://github.com/issues)?  Thanks.

**Comment by NitinIsFit** [comment]
Technical issues obviously.

**Reply by NitinIsFit** [reply]
Lmao, make a separate post for that, i am not a technician 😂

**Comment by queenofcorgis** [comment, marked as answer]
Hi there @rootsmusic - thanks for asking! It is about all things GitHub Issues (https://github.com/issues). For technical issues about GitHub we recommend posting those in the category related to your feedback and if there isn't a good fit use the General category!
